Directory.Packages.props

```diff
@@ -36,23 +37,24 @@
     <PackageVersion Include="Microsoft.Azure.SignalR" Version="1.27.0" />
     <PackageVersion Include="Microsoft.Extensions.Azure" Version="1.7.5" />
     <!-- Azure Management SDK for .NET dependencies -->
-    <PackageVersion Include="Azure.Provisioning" Version="0.3.0" />
-    <PackageVersion Include="Azure.Provisioning.AppConfiguration" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.ApplicationInsights" Version="0.1.0" />
-    <PackageVersion Include="Azure.Provisioning.CognitiveServices" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.CosmosDB" Version="0.1.0" />
-    <PackageVersion Include="Azure.Provisioning.EventHubs" Version="0.1.0" />
-    <PackageVersion Include="Azure.Provisioning.KeyVault" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.OperationalInsights" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.PostgreSql" Version="0.1.0" />
-    <PackageVersion Include="Azure.Provisioning.Redis" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.Resources" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.Search" Version="0.1.0" />
-    <PackageVersion Include="Azure.Provisioning.ServiceBus" Version="0.1.0" />
-    <PackageVersion Include="Azure.Provisioning.SignalR" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.Sql" Version="0.1.0" />
-    <PackageVersion Include="Azure.Provisioning.Storage" Version="0.2.0" />
-    <PackageVersion Include="Azure.Provisioning.WebPubSub" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning" Version="$(AzureProvisiongVersion)" />
+    <PackageVersion Include="Azure.Provisioning.AppConfiguration" Version="$(AzureProvisiongVersion)" />
+    <PackageVersion Include="Azure.Provisioning.ApplicationInsights" Version="1.0.0-alpha.20240912.1" />
```

Comment:
What is happening with this and operational insights?

Reply:
The pipeline was split into 2 becuase they couldn't publish all the packages at once. So they get different versions (for now).

Reply:
Let me guess ... the size of the generated YAML is too big and ADO is rejecting it :P Means the provisioning packages need to get moved into the sdk/[service] sub-folders instead of sdk/provisioning. At which point all versions will probably end up being different.

Reply:
Yep.  That's the plan - and long term, the versions will always be randomly released whenever different management plane libraries are updated.